src/pypwsqc/indicator_correlation.py

```diff
@@ -75,7 +90,24 @@ def indicator_distance_matrix(
 ):
     """Calculate indicator correlation and distance between reference and test stations.
 
-    return: indicator correlation and distance values
+    Parameters
+    ----------
+    da_a : xr.Dataset
```

Comment:
This has to be a xr.DataArray, i.e. it only contains one variable, which is the rainfall time series.